tests/e2e/lib.sh

```diff
@@ -353,13 +353,15 @@ deploy_optional_e2e_components() {
 
     if [[ "${INSTALL_COMPLIANCE_OPERATOR:-false}" == "true" ]]; then
         install_the_compliance_operator
+    elif [[ "${INSTALL_COMPLIANCE_OPERATOR_FROM_SOURCE:-false}" == "true" ]]; then
```

Comment:
nit: These should technically be mutually exclusive.

Reply:
Removed the from source install path since we're only installing from upstream indexes or downstream subscriptions for now.